src/web/src/license.ecpp

```diff
@@ -74,14 +74,18 @@
 //        fclose(src);
 //    }
     if ( accepted_license_version.empty() ) {
+        free (current_license_version); current_license_version = NULL;
         std::string reason = TRANSLATE_ME ("Please accept license first.");
         http_die("action-forbidden", "REST API call", reason.c_str ());
     }
     if ( !streq(accepted_license_version.c_str(), current_license_version) ) {
+        free (current_license_version); current_license_version = NULL;
         std::string msg = TRANSLATE_ME ("License version changed (new is %s, accepted is %s). Please accept new license first", current_license_version, accepted_license_version.c_str());
```

Comment:
This line uses `current_license_version`, free it afterwards please :)

Reply:
good catch